go-chaos/internal/k8helper.go

```diff
@@ -49,6 +49,26 @@ func CreateK8Client(kubeConfigPath string, namespace string) (K8Client, error) {
 }
 
 func createK8Client(settings KubernetesSettings) (K8Client, error) {
+	client, err := internalCreateClient(settings)
+	if err != nil {
+		return client, err
+	}
+
+	client.SaaSEnv, err = client.isSaaSEnvironment()
+	if err != nil {
+		return client, err
```

Comment:
:upside_down_face: Super nitpicky but I think we are not supposed to return the client in this case? 

```suggestion
		return nil, err
```

Reply:
Makes sense thanks 👍🏼